### PR TITLE
bugfix/AGCMD-2046 SSE Updates do not include data

### PR DESCRIPTION
**Pull request by jbeynar**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/agco/harvesterjs/pull/171%23discussion_r54104772%22%2C%20%22https%3A//github.com/agco/harvesterjs/pull/171%23discussion_r54108805%22%2C%20%22https%3A//github.com/agco/harvesterjs/pull/171%23issuecomment-189331844%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20f89ff460eb9091a79bdfdca2351ac0340bcc0352%20test/singleRouteSSE.spec.js%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/agco/harvesterjs/pull/171%23discussion_r54104772%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3F%20why%20this%20change%3F%22%2C%20%22created_at%22%3A%20%222016-02-25T14%3A47%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/372075%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ssebro%22%7D%7D%2C%20%7B%22body%22%3A%20%22With%20%601%60%20this%20test%20leve%20%60it%60%20too%20early%20and%20actully%20didn%27t%20test%20what%20it%20should%20test%22%2C%20%22created_at%22%3A%20%222016-02-25T15%3A10%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7689681%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jbeynar%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20test/singleRouteSSE.spec.js%3AL185-246%22%7D%2C%20%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/agco/harvesterjs/pull/171%23issuecomment-189331844%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Tests%20are%20passing%20on%20my%20env%20so%20I%27m%20merging%20this.%22%2C%20%22created_at%22%3A%20%222016-02-26T15%3A46%3A55Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/431064%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/blabno%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull f89ff460eb9091a79bdfdca2351ac0340bcc0352 test/singleRouteSSE.spec.js 5'></a> &nbsp;<b><a href='https://github.com/agco/harvesterjs/pull/171#discussion_r54104772'>File: test/singleRouteSSE.spec.js:L185-246</a></b>
- <a href='https://github.com/ssebro'></a> ? why this change?
- <a href='https://github.com/jbeynar'></a> With `1` this test leve `it` too early and actully didn't test what it should test
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/agco/harvesterjs/pull/171#issuecomment-189331844'>General Comment</a></b>
- <a href='https://github.com/blabno'></a> Tests are passing on my env so I'm merging this.

<a href='https://www.codereviewhub.com/agco/harvesterjs/pull/171?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/agco/harvesterjs/pull/171?approve=1'></a>&nbsp;<a href='https://github.com/agco/harvesterjs/pull/171'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by ssebro** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/agco/harvesterjs/pull/171'>Refresh</a>.<a href='#crh-update-none'></a>

File: test/singleRouteSSE.spec.js [hunk 1]
```diff
@@ -185,12 +185,61 @@ describe('EventSource implementation for resource changes', function () {
 
                     expect(_.omit(data, 'id')).to.deep.equal(payloads[counter].books[0]);
                     counter ++;
-                    if (counter === 1) {
+                    if (counter === 2) {
```
Comment (ssebro) [inline test/singleRouteSSE.spec.js]: ? why this change?

Comment (jbeynar) [inline test/singleRouteSSE.spec.js]: With `1` this test leve `it` too early and actully didn't test what it should test

**Comment by blabno** [comment]
Tests are passing on my env so I'm merging this.
